packages/liveblocks-zustand/src/index.ts

```diff
@@ -194,7 +194,7 @@ export function middleware<
         return;
       }
 
-      room = client.enter(roomId);
+      room = client.enter(roomId, { initialPresence: {} as TPresence });
```

Comment:
Zustand and Redux packages seem to get initialized differently, where the initial presence is expected always to be "empty" and immediately followed by a call to broadcastInitialPresence (which emits multiple presence chunk updates—one for every field). Not sure why that happens that way. Need to dig in there, but for now, let's just keep this working the way it currently works.